### DISCUSSION TITLE
Payload Product Roadmap

### DISCUSSION THREAD

**Discussion by denolfe** [payloadcms/payload#1539] [Announcement]
The Payload team uses GitHub discussions to plan our product roadmap. 

Check out the links below to see a list:

- [Roadmap P0](https://github.com/payloadcms/payload/discussions?discussions_q=label:%22roadmap:+p0%22+is:open) (Working  on or in queue)
- [Roadmap P1](https://github.com/payloadcms/payload/discussions?discussions_q=label:%22roadmap:+p1%22+is:open) 
- [Roadmap P2](https://github.com/payloadcms/payload/discussions?discussions_q=is:open+label:%22roadmap:+p2%22+)
- [Roadmap TBD](https://github.com/payloadcms/payload/discussions?discussions_q=label:%22roadmap:+TBD%22+is:open)

**Comment by siimsams** [comment]
Hey!

The roadmap links are broken.

**Reply by paburghelea** [reply]
Hi,

I think the labels were updated - like this: "roadmap-priority-1" to "roadmap:p1". Those links work:
[Priority 1](https://github.com/payloadcms/payload/discussions?discussions_q=label%3A%22roadmap%3A+p1%22+is%3Aopen)
[Priority 2](https://github.com/payloadcms/payload/discussions?discussions_q=label%3A%22roadmap%3A+p2%22+is%3Aopen)
[Priority 3](https://github.com/payloadcms/payload/discussions?discussions_q=label%3A%22roadmap%3A+p3%22+is%3Aopen)
